src/main/java/com/swiftype/appsearch/Client.java

```diff
@@ -279,13 +279,14 @@ public static String createSignedSearchKey(String apiKey, String apiKeyName, Map
         request.setHeader(HttpHeaders.CONTENT_TYPE, "application/json");
 
         if (reqBody != null) {
-          request.setEntity(new StringEntity(reqBody));
+          request.setEntity(new StringEntity(reqBody, "UTF-8"));
```

Comment:
`new StringEntity(reqBody, StandardCharsets.UTF_8)` is a tad safer, assuming we're requiring java 1.7 or higher

Reply:
@tehsven Is that actually a better option if we know it will require the user to be using 1.7 or higher? If you're worried about the magic string I could probably swap it out for this: https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/Consts.html#UTF_8. Thoughts?

Reply:
That seems fine as well. What is the min java version for this library? I didn't find it in the readme.

Reply:
@tehsven I don't actually know. I'd need to do some testing to figure that out.